ci/release/update-version.sh

```diff
@@ -37,60 +38,83 @@ function sed_runner() {
 
 # Centralized version file update
 echo "${NEXT_FULL_TAG}" > VERSION
+echo "${NEXT_FULL_TAG}" > RAPIDS_VERSION
```

Comment:
```suggestion
echo "${NEXT_FULL_TAG}" > ./VERSION
echo "${NEXT_FULL_TAG}" > ./RAPIDS_VERSION
```

Very very very minor comment, ignore if you want... I find it useful to add a leading `./` to stuff like this, to make it clear that we're piping to a file and not a shell variable.